CONTRIBUTING.md

```diff
@@ -0,0 +1,125 @@
+# How to contribute
+
+Contributions to ScalaBridge are welcomed and encouraged. Without input from volunteers we would have nothing. By participating in this project you agree to abide by the [Bridge Foundry](http://bridgefoundry.org/) [code of conduct](http://bridgefoundry.org/code-of-conduct/).
+
+You can contribute to ScalaBridge by either:
+
+1. Creating an `issue` to suggest an enhancement, identify a 'bug', or make another request, or
+2. Creating a `pull request` to submit revisions to existing content or create new content.  
+
+## Getting Started
+* To creating `issues` or making `pull requests` you need a [GitHub account](https://github.com).
+* To create `pull requests` follow *Setup* and *Pull Request Work Flow* instructions under the **Making Pull Request** section below.
+
+## Creating an Issue
+Creating an issue requires no technical expertise, just go [here](https://github.com/scalabridge/curriculum/issues/new), type a title and description for your issue and click `Submit new issue` when you are finished. A [Github help](https://help.github.com/articles/creating-an-issue/) page is also available.
+
+## Making Pull Requests
+
```

Comment:
Let's add a `tl;dr` section at the bottom of this for people that know git.

Something like:

* Create a fork 
* Use a branch
* Reference the issue (was this mentioned anywhere? If not, let's add a point about creating an issue if one doesn't exist and assigning the issue to yourself to help prevent duplicate work)
* Submit the pull request

Add a [header anchor](http://stackoverflow.com/a/16426829) to the section and link it here so people can skip over the details if they wish.